src/types.ts

```diff
@@ -0,0 +1 @@
+export type ReviewComment = { body: string; path: string; line: number };
```

Comment:
Consider using an interface instead of a type alias for `ReviewComment`. Interfaces are generally preferred for object types as they are more extensible and can be merged.